lib/validators/movie_filter.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const Joi = require('joi');
+
+const JoiReleaseYear = Joi.number().integer().min(1878).max(9999);
+
+module.exports = Joi.object().keys({
+  title: Joi.string().min(1).max(255).optional(),
+  title_contains: Joi.when(
+    'title',
+    {
```

Comment:
nit: just a styling comment on how we usually format `Joi.when`, you can see an example [here](https://github.com/lob/lob-api/blob/master/src/validators/users/update.js#L20)